### PR TITLE
added to k2-charts

### PR DESCRIPTION
**Pull request by jfelten**

### CODE REVIEW

File: gsbucketserver/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart to server contensts of a google bucket through http
```
Comment (coffeepac) [inline gsbucketserver/Chart.yaml]: spelling.  'contents', 'serve'

File: gsbucketserver/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# GS Bucket Server
+
+### The purpose of this chart is to serve up files in a Google Storage bucket through a web server.  Google Storage will allow you to expose the files through http over the internet, but not privately.  There is also limited control over how files are cached.  This chart gives more flexibility by setting up a private nginx server that servers up the files in your bucket.  This chart is designed to run in GKE which allows gcsfuse to work seemelssly with creating certificates or service accounts.  If not using GKE you will need to set that up yourself.
```
Comment (coffeepac) [inline gsbucketserver/README.md]: 'serves', not 'servers' 

note:  I made the same typo when making this comment

File: gsbucketserver/docker/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# To build this image:
+
+```bash
+docker build -t <YOUR_TAG> ./Dockerfile
+```
+
+## Docker image current lives at: jfelten/gsbucketserver
```
Comment (coffeepac) [inline gsbucketserver/docker/README.md]: currently

File: gsbucketserver/templates/NOTES.txt [hunk 4]
```diff
@@ -0,0 +1,17 @@
+This is the container is now serving the gs bucket: {{ .Values.gsbucketName }} over http
```
Comment (coffeepac) [inline gsbucketserver/templates/NOTES.txt]: This container is now serving...

File: gsbucketserver/templates/NOTES.txt [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+1. Get the application URL by running these commands:
+{{- if contains "NodePort" .Values.service.type }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "fullname" . }})
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT/login
+{{- else if contains "LoadBalancer" .Values.service.type }}
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get svc -w {{ template "fullname" . }}'
```
Comment (coffeepac) [inline gsbucketserver/templates/NOTES.txt]: ..the status by running ...

File: gsbucketserver/templates/deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+      containers:
+      - name: {{ .Chart.Name }}
+        command: ["/etc/scripts/start.sh"]
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        securityContext:
+          privileged: true
+        ports:
+        - containerPort: {{ .Values.service.internalPort }}
+#        livenessProbe:
```
Comment (coffeepac) [inline gsbucketserver/templates/deployment.yaml]: remove the commented out lines?

File: gsbucketserver/templates/deployment.yaml [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+#          httpGet:
+#            path: /
+#            port: {{ .Values.service.internalPort }}
+#        readinessProbe:
+#          httpGet:
+#            path: /
+#            port: {{ .Values.service.internalPort }}
+        volumeMounts:
+          # This name must match the volumes.name below.
+        - name: scripts
```
Comment (coffeepac) [inline gsbucketserver/templates/deployment.yaml]: if the name must match, why not make it a variable?

Comment (jfelten) [inline gsbucketserver/templates/deployment.yaml]: removed the comment - this was an artifact of cut and paste pasta

**Comment by coffeepac** [comment]
LGTM

**Pull request merged**